include/fuzzuf/utils/common.hpp

```diff
@@ -150,10 +150,10 @@ void CreateDir(std::string path);
 int OpenFile(std::string path, int flag);
 int OpenFile(std::string path, int flag, mode_t mode);
 
-void ReadFile(int fd, void *buf, u32 len, bool original_behaviour = true);
+ssize_t ReadFile(int fd, void *buf, u32 len, bool original_behaviour = true);
```

Comment:
```suggestion
ssize_t ReadFile(int fd, void *buf, u32 len, bool original_behaviour = false);
```

Reply:
This suggestion was not applied. Final decision is https://github.com/fuzzuf/fuzzuf/pull/53/commits/b8685960ab7d47c21f71f2f2901dd77e9aedea1c .